src/hud.cpp

```diff
@@ -87,24 +87,31 @@ Hud::Hud(video::IVideoDriver *driver, scene::ISceneManager* smgr,
 	m_halo_boxes.clear();
 
 	m_selection_pos = v3f(0.0, 0.0, 0.0);
-	std::string mode = g_settings->get("node_highlighting");
+	std::string mode_setting = g_settings->get("node_highlighting");
```

Comment:
I thought about this too, since it's in the constructor a const does not make a difference.